app/dashboard/static/js/app/view-builds-id.2020.5.js

```diff
@@ -603,19 +607,13 @@ require([
             spanNode = docFrag.appendChild(document.createElement('span'));
 
             tooltipNode = spanNode.appendChild(html.tooltip());
-            tooltipNode.setAttribute('title', 'Details for tree&nbsp;' + job);
 
-            aNode = tooltipNode.appendChild(document.createElement('a'));
-            str = '/job/';
-            str += job;
-            str += '/';
-            aNode.setAttribute('href', str);
-            aNode.appendChild(document.createTextNode(job));
+            aNode = tooltipNode.appendChild(document.createTextNode(job));
 
             spanNode.insertAdjacentHTML('beforeend', '&nbsp;&mdash;&nbsp;');
 
             tooltipNode = spanNode.appendChild(html.tooltip());
-            str = 'Test reports for tree';
+            str = 'Test reports for ';
```

Comment:
Add `« »` around the tree name to be consistent with other tooltips.

Reply:
I think we can keep the word "tree".  It would look like this:

"Test reports for tree «mainline»".